Movies-Selection/Movies-Selection/DataBase/DetailFilmRepository/DescriptionFilmDataStorage.swift

```diff
@@ -0,0 +1,31 @@
+// DescriptionFilmDataStorage.swift
+// Copyright © RoadMap. All rights reserved.
+
+import CoreData
+import Foundation
+/// DataStorage
+final class DescriptionFilmDataStorage {
```

Comment:
```suggestion
final class DescriptionFilmDataStorage {
```
подумай над 1 универсальным как у Матвея что б не плодить кучу стораджей